guides/common/assembly_managing-activation-keys.adoc

```diff
@@ -9,3 +9,13 @@ include::modules/proc_using-activation-keys-for-host-registration.adoc[leveloffs
 include::modules/proc_setting-the-service-level.adoc[leveloffset=+1]
 
 include::modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc[leveloffset=+1]
+
+include::modules/con_multiple-activation-keys-and-content-view-environments.adoc[leveloffset=+1]
+
+include::modules/proc_assigning-content-view-environments-to-activation-keys.adoc[leveloffset=+1]
+
+include::modules/proc_assigning-content-view-environments-to-an-activation-key-by-ID.adoc[leveloffset=+2]
```

Comment:
I don't really see much difference in the content of these two modules. The first one also seems to be using IDs unless I'm missing something. So can we merge this two modules into a single one, please?